<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerNameDialog</class>
 <widget class="QDialog" name="CustomerNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>220</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Enter customer name</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QRadioButton" name="radioButton_old">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Existing customer</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_new">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>New customer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>new customer name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
